contracts/bounties/BreakInvariantBounty.sol

```diff
@@ -9,16 +9,24 @@ import "../payment/PullPayment.sol";
  * @dev This bounty will pay out to a researcher if they break invariant logic of the contract.
  */
 contract BreakInvariantBounty is PullPayment, Ownable {
-  bool public claimed;
-  mapping(address => address) public researchers;
+  bool private claimed_;
+  mapping(address => address) private researchers;
 
   event TargetCreated(address createdAddress);
 
   /**
    * @dev Fallback function allowing the contract to receive funds, if they haven't already been claimed.
    */
   function() external payable {
-    require(!claimed);
+    require(!claimed_);
+  }
+
+  /**
+   * @dev Determine if the bounty was claimed.
+   * @return true if the bounty was claimed, false otherwise.
+   */
+  function wasClaimed() public view returns(bool) {
```

Comment:
@frangio what should I do here? Rename the function to `claimed` or rename the variable to `wasClaimed`?

Reply:
I like `claimed`. I'm not sure about `wasClaimed` because in English the correct thing would be "has been claimed".